.github/workflows/spellcheck.yml

```diff
@@ -0,0 +1,14 @@
+name: spellcheck
+on:
+  pull_request:
+jobs:
+  check-spelling:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Check Spelling
+        uses: rojopolis/spellcheck-github-actions@0.33.1
+        with:
+          config_path: .github/spellcheck-settings.yml
+          task_name: Markdown
```

Comment:
Does it have to be in it's on file? Why not keep it in `linters.yml`?

Reply:
maintainability - it still reports back to GH.